CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+##2015-02-09 - Supported Release 3.2.0
+###Summary
+This release includes several new features and bugfixes, including support for various plugins, making the output from mysql_password more consistent when input is empty and improved username validation.
+
+####Features
+- Add type and provider to manage plugins
+- Add support for authentication plugins
+- Add support for mysql_install_db on freebsd
+- Add `create_root_user` and `create_root_my_cnf` parameters to `mysql::server`
+
+####Bugfixes
+- Remove dependency on stdlib >= 4.1.0 (MODULES-1759)
```

Comment:
The correct dependency in the metadata wasn't there, but the actual dependency (function from 4.1.0) was.